raw_data/preprocessed/africa/zanzibar_english.txt

```diff
@@ -260,6 +260,8 @@ PART 9 - MISCELLANEOUS PROVISIONS
 
 (122) The Removal of Natural Produce Rules, 1984, L.N. 21 of 1984, are repealed.
 
-(123) [1] The following laws are amended as set out in Schedule 5: [a] The District and Town Councils Act, 1995, Act No. 4 of 1995; and [b] The Zanzibar Municipal Council Act, 1995, Act No. 3 of 1995. [2] The Commission for Lands and Environment Act, 1989 Act No. 6 shall be amended accordingly pursuant to the provisions of this act.
+(123) [1] The following laws are amended as set out in Schedule 5: [a] The District and Town Councils Act, 1995, Act No. 4 of 1995; and [b] The Zanzibar Municipal Council Act, 1995, Act No. 3 of 1995. [2] The Commission for Lands and Environment Act, 198
+
+PART 9 - Act No. 6 shall be amended accordingly pursuant to the provisions of this act.
```

Comment:
These lines need to be recombined into article 123; the previous state was correct.